test/fixtures/select_expressions.ftl

```diff
@@ -34,3 +34,8 @@ nested-variant-list =
           *[two] Value
        }
     }
+
+# ERROR Missing line end after variant list
+missing-line-end =
+    { 1 ->
+        *[one] One}
```

Comment:
This is still a syntax error because any list of variants _must_ end with a `line_end` to be well-formed.